CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # TAMR AWS EMR module
 
+## v6.1.0 - August 4th, 2021
+
+* Adds new variable `abac_valid_tags` to be used in IAM Policies conditions for creating EMR Resources using ABAC
```

Comment:
Since the policy does change quite a bit, does upgrading from version v6.0.0 go fairly smoothly? Otherwise we may want to mark this as a major version.

Reply:
They are safe since policy updates are done in-place. The new policies won't break existing clusters as all "old" statements were either kept or merged into the new ones. Also, in case the user doesn't pass `abac_valid_tags` conditions won't be created. 
The one problem I see happening here is if a user pass a map that is too long and that would exceed the limit of chars in a policy. 
I haven't tested the situation of breaking like this in an update but I think AWS API just wouldn't perform the update and keep the current policy working.
We can test those cases before proceeding just to be extra careful.